docs/source/tutorial/convnet.rst

```diff
@@ -107,11 +105,20 @@ computations in the model. Just prepare the data, then give it to the model.
 The way this works is the resulting output :class:`~chainer.Variable` from the
 forward computation has a :meth:`~chainer.Variable.backward` method to perform
 autograd. In the above model, :meth:`__call__` has a ``if`` statement at the
-end to switch its behavior by the model's running mode, i.e., training mode or
-not. When it's in training mode, this method returns the output value of the
+end to switch its behavior by the Chainer's running mode, i.e., training mode or
+not. Chainer presents the running mode as a global variable `chainer.config.train`.
+When it's in training mode, this method returns the output value of the
```

Comment:
How about changing the pronoun "this method" to the explicit expression i.e. "`__call__`"?
It has become far away from the previous appearance.